transport/transport.go

```diff
@@ -303,6 +303,24 @@ func StreamFromContext(ctx context.Context) (s *Stream, ok bool) {
 	return
 }
 
+// Peer connection info
+type Peer struct {
```

Comment:
understood. Let me think about it. Let service handler call transport.PeerFromContext is unpleasant...